### PR TITLE
update(userspace): minor pre-release improvements

### PR DESCRIPTION
**Pull request by jasondellaluce**
**What type of PR is this?**

/kind cleanup

**Any specific area of the project related to this PR?**

/area engine

**What this PR does / why we need it**:

This adds a bunch of chores and minor changes to clean-up the code for the upcoming release. Goals are:
- Make log messages more uniform and consistent in formatting
- Minor performance improvement chores: passing string by reference, using unordered_set, etc...

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
update(userspace): minor pre-release improvements
```

### CODE REVIEW

File: userspace/falco/app_actions/open_inspector.cpp [hunk 1]
```diff
@@ -111,7 +111,7 @@ application::run_result application::open_live_inspector(
 				falco_logger::log(LOG_INFO, "Trying to inject the Kernel module and starting the capture again...");
```
Comment (Andreagit97) [inline userspace/falco/app_actions/open_inspector.cpp:111]:
```suggestion
				falco_logger::log(LOG_INFO, "Trying to inject the Kernel module and opening the capture again...");
```

**Review by Andreagit97 — DISMISSED** [review]
/approve

**Review by Andreagit97 — APPROVED** [review]
/approve

**Review by FedeDP — APPROVED** [review]
/approve

**Pull request merged**